experiments/AMIP/modular/components/flux_calculator.jl

```diff
@@ -12,7 +12,7 @@ of the lowest level.
 function set_ρ_sfc!(ρ_sfc, T_S, integrator)
     ts = integrator.p.ᶜts
     thermo_params = CAP.thermodynamics_params(integrator.p.params)
-    ts_int = Spaces.level(ts, 1)
+    ts_int = ClimaCore.Spaces.level(ts, 1)
```

Comment:
Being that there are several key modules in ClimaCore, it may be easier (result in a smaller diff, too) to add `import ClimaCore.Spaces as Spaces` to the top of wherever this is included.